dht/server.go

```diff
@@ -331,6 +359,9 @@ func (s *Server) getNode(addr dHTAddr, id string) (n *node) {
 	if len(s.nodes) >= maxNodes {
 		return
 	}
+	// (@cathalgarvey) DHT Security is supposed to prevent insecure nodes from
+	// hosting data: not to exclude them from DHT routing entirely. Does
+	// this check exclude them entirely from the local routing table?
```

Comment:
Yes it does. You can put a comment here saying that's the intended effect for the next guy if you like.